src/FormsyCheckbox.jsx

```diff
@@ -22,12 +22,17 @@ let FormsyCheckbox = React.createClass({
   _setMuiComponentAndMaybeFocus: _setMuiComponentAndMaybeFocus,
 
   render: function () {
+    var _value = this.getValue();
+    var defaultChecked = this.props.defaultChecked;
+    if(typeof _value  === 'undefined' )
```

Comment:
Please remove the extra space and put in braces for this, and remove the underscore dangle.

Thanks.